pkg/container/container_test.go

```diff
@@ -68,6 +69,18 @@ var _ = Describe("the container", func() {
 				Expect(len(containers) > 0).To(BeTrue())
 			})
 		})
+		When(`listing containers with the "include restart" option`, func() {
+			It("should return both stopped, restarting and running containers", func() {
+				client = dockerClient{
+					api:               docker,
+					pullImages:        false,
+					includeRestarting: true,
+				}
+				containers, err := client.ListContainers(filters.NoFilter)
+				Expect(err).NotTo(HaveOccurred())
+				Expect(len(containers) > 0).To(BeTrue())
+			})
+		})
```

Comment:
could you please add a test case for the opposite as well? not including restarted when the flag is not set, that is.

Reply:
Sure 👍